### PR TITLE
Set https://localhost default identity server

### LINKED ISSUE #26: Allow IS that is not responding
Two IS are hard coded vector.im and riot.im: https://github.com/vector-im/riot-android/blob/047e156cc7f3c708fdcdb4e90c0df63155076bae/vector/src/main/java/im/vector/receiver/VectorRegistrationReceiver.java#L49

Is it possible to allow any host, even localhost? It would fail to connect but that shouldn't prevent using the client. I've tested it by adding both vector.im and riot.im to Blokada blocked hosts.

### PR DESCRIPTION
**Pull request by afonari**
closes #26 

Main changes are in:
vector/src/main/java/im/vector/activity/LoginActivity.java 
and
vector/src/main/res/values/config.xml

This is PR for https://github.com/LiMium/mini-vector-android/issues/26

Default login screen looks like:
mini-matrix-screenshot

### CODE REVIEW

**Comment by hrj** [comment]
Awesome! I didn't know there was a travis-ci configuration file in this repo! I have enabled Travis-CI for this repo now. The next commit should trigger a build.

File: .travis.yml [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-    # use the latest revision of Android SDK Tools
     - tools
     - platform-tools
-
-    # The BuildTools version used by your project
-    - build-tools-28.0.3
-
-    # The SDK version used to compile your project
-    - android-28
+    - tools
```
Comment (hrj) [inline .travis.yml]: `tools` is listed twice.

Minor, but pushing a commit will help trigger a travis-ci build.

File: .travis.yml [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+ 
+deploy:
+  provider: releases
+  file: "vector/build/outputs/apk/app/debug/vector-app-debug.apk"
+  skip_cleanup: true
+  draft: true
+  on:
+    branch: patch-1
+  api_key:
+    secure: "qh/v61K0ZP/C29FG57JC3sPeVlJw3D2VXA+M5sym+GMR5mOQd3RkofB4kdSwma0zj6AiEVaTBZ+VWnadLBAOW9avz7BqS2MUn8EbU8dAunwuHiyoMwfq4hPofolt7ftpOO5PaBb3pBYcJdld/94ai+yix11IpxShvM/li5AS+XnAYYne4NQ82GedftPl4pdMsbtla0CBD3lmg1GZzJNALZUvvoxE9lXwkRfPT7bq6pGM+yHkCrCYpIjIyuSCGi5bV6d5uAz8XPXL6C20Gd/kaI47OrkyMvVMZJwCQMSlycwOtiPIv6hWX4v8yb1NQ7NtlHOREM/JwMaDhyeCLwLP+uGDfshA96xD+9z4glySYbxn50orc/DRD6QkfBJW219KTcMRHtueR7VZZl0K1vMjmEzioP6mabyrsZE96fxxJAMBfwgsrozb/gHtHRimoHcs3R4rNu8CLgjolKTesdkhSvbSjasUotKqP9OXE+8kSqojToRQurmnRqcYTpORlOi1/8DTaEoXZu4fDKgOwod2N3fvxK5sxp66yF9yWC+76yR0UMU8sL/ZXVUoO5q3Duteqqk5GytQkUREsQO46cPaUEJY3/tBS9YCeTcPiaM4veOaWZGEkds4jPbromUpa/GFhj8/vEmC4QYZ/U/u9tU8a7IgDwqd5GaWDCuGED5YvfY="
```
Comment (hrj) [inline .travis.yml:53]: What key is this and where is this getting deployed?

Comment (afonari) [reply .travis.yml:53]: From https://medium.com/@harmittaa/travis-ci-android-example-357f6e632fc4

You can generate the API key by going to your account settings on GitHub, Personal access tokens, Generate new token. Set the scope to public_repo and generate the token. Remember to copy the access token.

Encrypt the API key (you will need to install travis gem locally: gem install travis) and add it to the deploy section under api_key.

```travis encrypt apikey```

File: .travis.yml (hunk 1 above)
Comment (afonari) [reply .travis.yml]: Now it's listed only once.

File: .travis.yml (hunk 2 above)
Comment (hrj) [reply .travis.yml:53]: Cool. I will merge this PR, but I am going to knock out the deploy part, as I am not comfortable putting keys out there in the cloud.

**Comment by afonari** [comment]
Sounds good, thanks.

**Comment by Bubu** [comment]
This is now finally available in the f-droid release.
It'skinda weird when using a custom homeserver, as the server lookup logic overrides this again with the server url set by the homeserver. Which I guess makes sense, but this makes this default only useful when using a matrix.org account I guess?

**Comment by hrj** [comment]
@Bubu Yeah this is very hacky at the moment. We should be able to reverse this (without compromising privacy) once upstream makes IS optional.

I am creating a new issue now to remind us.

**Pull request merged**